src/dag-node.js

```diff
@@ -137,18 +139,26 @@ class DAGNode {
   /*
    * multihash - returns the multihash value of this DAGNode
    */
-  multihash (callback) {
-    if (!this.cached || this._updated) {
-      util.serialize(this, (err, serialized) => {
+  multihash (type, callback) {
+    if (typeof type === 'function') {
+      callback = type
+      type = 'sha2-256'
+    }
+    if (!this._cached[type] || this._updated) {
+      waterfall([
+        (cb) => util.serialize(this, cb),
+        (serialized, cb) => util.hash(type, serialized, cb)
+      ], (err, digest) => {
```

Comment:
Calling it a `digest`  can be misleading, because it is a multihash (it is prefixed with the multicodec)